cloud_images/centos7/dcos_vol_setup.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
 set -o errexit -o nounset -o pipefail
 
+# Avoid getting killed due to https://bugs.freedesktop.org/show_bug.cgi?id=84923 if
+# systemd-journald is restarted while we are running.
+trap 'echo dodged SIGPIPE' PIPE
```

Comment:
If we get a SIGPIPE it means the reader is gone. Adding the `echo` instead of just `''` shouldn't give any extra output, just another SIGPIPE.